src/shared/ui/survey/RadioActivityItem/RadioActivityItem.tsx

```diff
@@ -36,31 +34,25 @@ const RadioActivityItem: FC<RadioActivityItemProps> = ({
   }, [randomizeOptions]);
 
   const onValueChange = (value: string) => {
-    setRadioGroupValue(value);
     onChange(value);
   };
 
   return (
     <YStack>
       <RadioGroup
-        value={radioGroupValue}
+        value={initialValue ?? ''}
```

Comment:
looks weird, why RadioGroup gets initialValue eq. to empty string ?

Reply:
Okay RadioGroup accepts either string or undefined. Null is not acceptable unfortunately, so I will undefined in this case